### PR TITLE
Update routing.rst

### PR DESCRIPTION
**Pull request by DurandSacha**
add some uses statements ( InputInterface & OutputInterface )

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -1946,6 +1946,8 @@ This information can be configured per command too::
     // src/Command/SomeCommand.php
     namespace App\Command;
 
+    use Symfony\Component\Console\Input\InputInterface;
+    use Symfony\Component\Console\Output\OutputInterface;
     use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
     use Symfony\Component\Routing\RouterInterface;
     // ...
```
Comment (HeahDude) [inline routing.rst]: This can be removed though

**Review by HeahDude — CHANGES_REQUESTED** [review]
Thanks for that PR, a minor comment

File: routing.rst (hunk 1 above)
Comment (HeahDude) [reply routing.rst]: This should still be removed :)

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Sacha.

**Pull request merged**